.eslintrc.json

```diff
@@ -42,6 +42,12 @@
           "ClassDeclaration": true
         }
       }
-    ]
+    ],
+    "prefer-destructuring":"off",
+    "no-console":"off",
+    "no-unused-expressions": "off",
+    "@typescript-eslint/no-explicit-any": "off",
```

Comment:
This rule ensures that typescript is used correctly, so its not ideal to remove it
```suggestion
```